discovery/hetzner/hcloud.go

```diff
@@ -124,8 +125,12 @@ func (d *hcloudDiscovery) refresh(ctx context.Context) ([]*targetgroup.Group, er
 			}
 		}
 		for labelKey, labelValue := range server.Labels {
-			label := model.LabelName(hetznerLabelHcloudLabel + strutil.SanitizeLabelName(labelKey))
-			labels[label] = model.LabelValue(labelValue)
+			presentLabel := model.LabelName(hetznerLabelHcloudLabelPresent + strutil.SanitizeLabelName(labelKey))
+			labels[presentLabel] = model.LabelValue("true")
+			if labelValue == "" {
+				label := model.LabelName(hetznerLabelHcloudLabel + strutil.SanitizeLabelName(labelKey))
```

Comment:
this if should go away. We should have the label for every hcloud label.

Reply:
This doesn't make sense, why should the label be there if prometheus skips it at all?

Reply:
ooh it is the normal label :)

Any way works, I just prefer as less `if` as possible.